hat/assets/js/apps/Iaso/domains/dataSources/components/DataSourceDialogComponent.js

```diff
@@ -105,6 +105,12 @@ const formIsValid = form => {
     );
 };
 
+const translateErrors = (dhis2Credentials, formatMessage) => {
+    return dhis2Credentials?.errors.length > 0
+        ? [formatMessage(MESSAGES[dhis2Credentials?.errors])]
```

Comment:
This will error if `dhis2Credentials.errors` is not in `MESSAGES`.

Also: dhis2Credentials.errors is a string? Is there always only one error, or can we have a string of comma-separated errors, like: `"error1,error2"`? In this case we need to be able to handle it.

Reply:
@quang-le in the backend I return only a defined string key which won't contains a comma as you can see in the screenshot. 
![Screenshot from 2023-09-12 12-47-51](https://github.com/BLSQ/iaso/assets/19631540/007c3d53-aaca-49d7-82da-0c54858b047b)